API/classes/youtube/youtube_channel_id.py

```diff
@@ -62,14 +62,14 @@ def re_channel_from_url(self, url):
+        regex = r"(?:(?<=channel/)|(?<=c/)|(?<=user/))[^/?\s]+"
```

Comment:
Awesome, that is much more readable and maintainable!

Reply:
@Travis-Owens your welcome!
sorry for the previous pull request I didn't know what exactly you want and I just add **c** for channel name